service.upnext/addon.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version='1.0' encoding='UTF-8' standalone='yes'?>
+<addon id="service.upnext" name="Up Next" provider-name="im85288" version="1.0.0">
+    <requires>
+        <import addon="xbmc.python" version="2.26.0"/>
+        <import addon="script.module.requests" version="2.9.1" />
+        <import addon="script.module.simplejson" version="3.3.0"/>
+    </requires>
+    <extension library="service.py" point="xbmc.service" start="login"/>
```

Comment:
Please remove `start="login"` as this was removed for leia. It will always start on login by default now.